source/includes/scripting/fact-require-ems-modules.rst

```diff
@@ -0,0 +1,24 @@
+There are two packaging standards for Node.js modules.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Packaging Standard
+     - Works with require()
+
+   * - ``CommonJS`` (CJS)
+     -  Yes
+
+   * - ``ECMAScript``
```

Comment:
I’d call this “ES Module” or if you really want to use the full name “ECMAScript Module” (that’s also in line with the Node.js docs linked below)